### DISCUSSION TITLE
ACL in cloudforet is needed

### DISCUSSION THREAD

**Discussion by gikang82** [cloudforet-io/community#7] [Ideas]
In some domain, User needs acl(access control list) to prohibit console access from out side of the location(company or home).
Maybe it might be available to give access from physical loadbalancer or firewall... I  think it is more powerful for cloudforet to control them through administration function.

**Comment by whdalsrnt** [comment]
hello. @gikang82 
Thank you very much for suggesting good features.

As Cloudforet's project grows, security becomes very important.
Several security features will be developed on the roadmap for 2023, and Network ACL settings for each domain will also be released.

**Comment by whdalsrnt** [comment]
This item is linked in the issue below.
- https://github.com/cloudforet-io/community/issues/9
